src/scss/includes/placeIcon.scss

```diff
@@ -2,7 +2,7 @@
   display: block;
   font-size: 32px;
   border-radius: 50%;
-  width: 36px;
+  width: 34px;
```

Comment:
Another change that seems very arbitrary. And as a result, the favorite icon in your screenshots looks compressed.